#Expyriment 0.9.0 (Python 2.7.11), .xpe-file, coding: cp1255
#date: Wed Apr 11 2018 12:28:22
#sha1: e5c5e8
#modules: grid.py (68c27b), writeToExcel.py (93ffb1), nback.py (e7d68d), runStroopTask.py (7d9a32), stroop.py (6289c4), runNbackTask.py (4eb18a), restBlock.py (9d51ab)
#display: size=(800, 600), window_mode=True, open_gl=2
#os: ('Windows', 'DESKTOP-A6JHBFQ', '10', '10.0.16299', 'AMD64', 'Intel64 Family 6 Model 142 Stepping 9, GenuineIntel')
Time,Type,Event,Value,Detail,Detail2
441,design,log,
#design: #exp: main
#design: block_cnt,block_id,trial_cnt,trial_id
441,design,logged,
441,Experiment,started
466,Stimulus,presented,0
957,Mouse,received,0-False,wait_event
982,Stimulus,presented,7
2868,Mouse,received,0-False,wait_event
3460,Mouse,received,0-False,wait_event
3474,Stimulus,presented,14
4348,Mouse,received,0-False,wait_event
4364,Stimulus,presented,21
5524,Mouse,received,0-False,wait_event
5525,stderr,received,'Traceback (most recent call last):'
5525,stderr,received,'  File "C:/Users/NOA/fMRI_E_Project/BasicTasks/main.py", line 80, in <module>'
5526,stderr,received,'    start_new_exp = nBack.start_run();'
5526,stderr,received,'  File "C:\\Users\\NOA\\fMRI_E_Project\\BasicTasks\\runNbackTask.py", line 80, in start_run'
5526,stderr,received,'    start_new_exp = self.run_blocks();'
5526,stderr,received,'  File "C:\\Users\\NOA\\fMRI_E_Project\\BasicTasks\\runNbackTask.py", line 491, in run_blocks'
5527,stderr,received,"    instructions_path, self.experiment.exp, block_type = 'p')"
5527,stderr,received,"TypeError: __init__() got an unexpected keyword argument 'block_type'"
